src/XamlStyler.dotUltimate/src/rider/main/resources/META-INF/pluginIcon.svg

```diff
@@ -0,0 +1,40 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="300" height="300" viewBox="0 0 300 300">
```

Comment:
It looks like these were inadvertently converted from RGB to CYMK -- any chance we can adjust that back (colors are off).

Reply:
Fixed.